backend/src/main/resources/application.yaml

```diff
@@ -11,4 +11,16 @@ spring:
         default_schema: simple_report
   liquibase:
     default-schema: ${spring.jpa.properties.hibernate.default_schema}
+graphql:
```

Comment:
You appear to have found the auto-configuration that I wasn't able to figure out last week–yay!